From 3dc8a678a844247d0afcf7e30fb3cbd5ccbd828f Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 29 Jun 2017 14:05:06 -0700
Subject: [PATCH] Rename conflicting variable 'mips'

Fixes
test/srtp_driver.c:344:12: error: expected identifier or '(' before numeric constant
     double mips = mips_estimate(1000000000, &ignore);

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 test/srtp_driver.c | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/test/srtp_driver.c b/test/srtp_driver.c
index 8872971..d38ea19 100644
--- a/test/srtp_driver.c
+++ b/test/srtp_driver.c
@@ -341,7 +341,7 @@ main (int argc, char *argv[]) {
   if (do_codec_timing) {
     srtp_policy_t policy;
     int ignore;
-    double mips = mips_estimate(1000000000, &ignore);
+    double est = mips_estimate(1000000000, &ignore);
 
     crypto_policy_set_rtp_default(&policy.rtp);
     crypto_policy_set_rtcp_default(&policy.rtcp);
@@ -353,33 +353,33 @@ main (int argc, char *argv[]) {
     policy.allow_repeat_tx = 0;
     policy.next = NULL;
 
-    printf("mips estimate: %e\n", mips);
+    printf("mips estimate: %e\n", est);
 
     printf("testing srtp processing time for voice codecs:\n");
     printf("codec\t\tlength (octets)\t\tsrtp instructions/second\n");
     printf("G.711\t\t%d\t\t\t%e\n", 80, 
-           (double) mips * (80 * 8) / 
+           (double) est * (80 * 8) / 
 	   srtp_bits_per_second(80, &policy) / .01 );
     printf("G.711\t\t%d\t\t\t%e\n", 160, 
-           (double) mips * (160 * 8) / 
+           (double) est * (160 * 8) / 
 	   srtp_bits_per_second(160, &policy) / .02);
     printf("G.726-32\t%d\t\t\t%e\n", 40, 
-           (double) mips * (40 * 8) / 
+           (double) est * (40 * 8) / 
 	   srtp_bits_per_second(40, &policy) / .01 );
     printf("G.726-32\t%d\t\t\t%e\n", 80, 
-           (double) mips * (80 * 8) / 
+           (double) est * (80 * 8) / 
 	   srtp_bits_per_second(80, &policy) / .02);
     printf("G.729\t\t%d\t\t\t%e\n", 10, 
-           (double) mips * (10 * 8) / 
+           (double) est * (10 * 8) / 
 	   srtp_bits_per_second(10, &policy) / .01 );
     printf("G.729\t\t%d\t\t\t%e\n", 20, 
-           (double) mips * (20 * 8) /
+           (double) est * (20 * 8) /
 	   srtp_bits_per_second(20, &policy) / .02 );
     printf("Wideband\t%d\t\t\t%e\n", 320, 
-           (double) mips * (320 * 8) /
+           (double) est * (320 * 8) /
 	   srtp_bits_per_second(320, &policy) / .01 );
     printf("Wideband\t%d\t\t\t%e\n", 640, 
-           (double) mips * (640 * 8) /
+           (double) est * (640 * 8) /
 	   srtp_bits_per_second(640, &policy) / .02 );
   }
 
-- 
2.13.2

